.github/.wordlist.txt

```diff
@@ -830,6 +832,7 @@ MLD
 mmevk
 moal
 ModeSelect
+ModelCommand
```

Comment:
ModelCommand is not a word, others need changes as well (GetWaitDuration is not a word either).